packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -68,6 +68,12 @@ class Ticker {
 
   TickerFuture _future;
 
+  /// Whether the platform is requesting that animations be disabled.
+  ///
+  /// See also:
+  ///   [AccessibilityFeatures.disableAnimations], for the setting this value comes from.
```

Comment:
Should either have a bullet or be all one sentence:

```dart
  /// See also:
  ///
  ///  * [AccessibilityFeatures.disableAnimations], for the setting this value comes from.
```

Reply:
Done